src/main/java/user_interface/custom_game/custom_game_file_management/CustomGameFileManager.java

```diff
@@ -0,0 +1,6 @@
+package user_interface.custom_game.custom_game_file_management;
+
+import use_cases.custom_game.custom_game_inner_file_management.ICustomGameFileManager;
```

Comment:
The user interface layer shouldn't directly access use cases layer

Reply:
I was aiming for something like this 
![image](https://user-images.githubusercontent.com/113129102/202204148-834cf042-c27b-47b8-bf97-307d3a3c9578.png)

It needs the import to access its interface

Reply:
I see, in that case it should be fine. Sorry!

Reply:
I see. In that case it should be fine, sorry!